tests/foreman/cli/test_location.py

```diff
@@ -407,3 +408,29 @@ def test_negative_update_parent_with_child(self, request):
             Location.update({'id': location['id'], 'parent-id': location['id']})
         location = Location.info({'id': location['id']})
         assert location['parent'] == parent_location['name']
+
+    @pytest.mark.skip_if_open("BZ:1937009")
+    @pytest.mark.tier1
+    def test_positive_nested_location(self, request, module_org):
+        """View nested location in an organization
+
+        :id: cce4a33d-7743-47ef-b437-2539fdc73c44
+
+        :customerscenario: true
+
+        :BZ: 1937009
+
+        :expectedresults: Nested location can be viewed
+
+        :CaseImportance: Medium
+        """
+        parent_location = _location(request, {'organization-id': module_org.id})
+        location = _location(
+            request, {'parent-id': parent_location['id'], 'organization-id': module_org.id}
+        )
+        Org.add_location({'location-id': parent_location['id'], 'name': module_org.name})
+        Org.add_location({'location-id': location['id'], 'name': module_org.name})
```

Comment:
If I'm reading the associated bugzilla right, then the child location is expected to inherit the org association form the parent. If that is true, the two lines above shouldn't be necessary.

Reply:
@pondrejk Unless we add the locations to the organization, it doesn't show in the organization which is associated to it.

Reply:
Yes, but isn't that a result of the bug in 1937009? If we want to test the desired behavior mentioned in the bug, I think we shouldn't add the location to the org. The test shouldn't pass now, because the bz is in new state. That is all right because you use skip_if_open. Once the bug is fixed, the test will run and should pass...

Reply:
Yes, I overlooked it. Updated the same.